packages/core-mobile/babel.config.js

```diff
@@ -14,13 +14,12 @@ module.exports = {
           '.ios.js',
           '.ios.tsx'
         ],
-        root: ['./app'],
+        root: ['./app', './src'],
```

Comment:
looks like you have app nested inside src. can we have only one of them?

Reply:
moved the expo-router app root to `/app/new` and revered babel configuration.